Automattic/vip-design-system
# Format all files to avoid future PR interruptions

## Description

Fixing formatting issues with prettier and eslint. Using the correct VIP formatter.

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook. All components should continue to work.

This is mainly a formatting PR.
